src/modes/map/map_mode.cpp

```diff
@@ -877,9 +881,10 @@ void MapMode::StartEnemyEncounter(EnemySprite* enemy, bool hero_init_boost, bool
 
     const std::vector<BattleEnemyInfo>& enemy_party = enemy->RetrieveRandomParty();
     for(uint32_t i = 0; i < enemy_party.size(); ++i) {
-        BM->AddEnemy(enemy_party[i].enemy_id,
-                     enemy_party[i].position_x,
-                     enemy_party[i].position_y);
+        auto enemy_party_i = enemy_party[i];
```

Comment:
I'd rather call this `current_enemy` for instance, the `_i` would not be part of my practices. Could you change it? Thanks!